app/controllers/patients_controller.rb

```diff
@@ -242,16 +242,9 @@ def update
     # Reset symptom onset date if moving from isolation to exposure
     reset_symptom_onset(content, patient, :system) if !content[:isolation].nil? && !content[:isolation]
 
-    # Grab diff, attempt to update, else return to index if failed
+    # Update patient history with detailed edit diff
     patient_before = patient.dup
-    if patient.update(content)
-      diffs = patient_diff(patient_before, patient)
-      unless diffs.length.zero?
-        pretty_diff = diffs.collect { |d| "#{d[:attribute].to_s.humanize} (\"#{d[:before]}\" to \"#{d[:after]}\")" }
-        comment = "User edited a monitoree record. Changes were: #{pretty_diff.join(', ')}."
-        history = History.record_edit(patient: patient, created_by: current_user.email, comment: comment)
-      end
-    end
+    Patient.detailed_history_edit(patient_before, patient, current_user, allowed_params) if patient.update(content)
```

Comment:
When I updated an existing user with the UI, I got a history item that looked like:
![image](https://user-images.githubusercontent.com/52747882/96776023-6ab2a400-13b6-11eb-8c70-33c6c05953ff.png)

I think that is because `current_user` should be `current_user.email`.

Reply:
Yes, I didn't notice the wonky email label when testing -good catch! Fixed.